src/common/Search/index.js

```diff
@@ -0,0 +1,42 @@
+
+import { useLocation } from "react-router-dom";
+import { StyledSearch } from "./styled";
+import { useReplaceQueryParameter, useQueryParameter } from "./queryParameterHooks";
+import { queryParameters } from "./queryParameters";
+
+export const Search = () => {
+  const location = useLocation();
+  const query = useQueryParameter(queryParameters.search);
+  const replaceQueryParameter = useReplaceQueryParameter();
+
+  const onInputChange = ({ target }) => {
+    replaceQueryParameter([
+      {
+        key: queryParameters.search,
+        value: target.value.trim(),
```

Comment:
This is the culprit of the search problem - you can't add a space, because it is trimmed immediately :(

Reply:
I see that value was `target.value.trim() !== "" ? target.value : undefined` in previous commits, let's get back to that :D